go/vt/vtgate/logstats/logstats.go

```diff
@@ -155,9 +156,9 @@ func (stats *LogStats) Logf(w io.Writer, params url.Values) error {
 	var fmtString string
 	switch *streamlog.QueryLogFormat {
 	case streamlog.QueryLogFormatText:
-		fmtString = "%v\t%v\t%v\t'%v'\t'%v'\t%v\t%v\t%.6f\t%.6f\t%.6f\t%.6f\t%v\t%q\t%v\t%v\t%v\t%q\t%q\t%q\t%q\t%q\t%v\t%v\n"
+		fmtString = "%v\t%v\t%v\t'%v'\t'%v'\t%v\t%v\t%.6f\t%.6f\t%.6f\t%.6f\t%v\t%q\t%v\t%v\t%v\t%q\t%q\t%q\t%q\t%q\t%v\t%v\t\"%v\"\n"
```

Comment:
Do we need %v in quotes, The other places seem to have them in single quotes or not at all

Reply:
good point

Reply:
I've addressed the feedback. Note that the output format does not change, so \"%v\" is in fact equivalent to %q but %q is more compact.